### DISCUSSION TITLE
119

### DISCUSSION THREAD

**Discussion by giscus** [CODe5753/giscus#3] [Comments]
# 119

서론 2023년 12월, 카카오뱅크 결제코어개발팀 백엔드 개발자로 합류하게 되었습니다. 넥슨 코리아 퇴사와 동시에 쉬는 기간 없이 입사를 결정했고, 이 아티클이 등록되는 시점엔 아마 출근을 앞두고 있을 겁니다. "하반기 이직을 꼭 해야겠다!"라는 목표보다는 경력 이력서 기반으로 어떤 질문이 들어올지 경험을 쌓기 위한 목표가 컸습니다. 그래서 하반기 목표는 "기술 면접 감 잡기"였습니다. 그러나, 진짜 문제는 채용 공고가 안 떴다는 점이었습니다.. 정말 다행스럽게도 스터디를 병행하던 타이밍에 딱 2개의 공고를 확인할 수 있었고, 해당 공고에 망설임 없이 지원할 수 있었습니다. 글을 작성하는 지금도 총 10개의 회사 채용 공고를 즐겨찾기 해두었는데, 제가 지원 가능한 채용 공고가 단 하나도 없네요.. 본론으..

https://imksh.com/119?category=965539

**Comment by Denia-park** [comment]
정보 잘 보고 갑니다! 

열심히 하신게 느껴져서 보기 좋습니다!

**Reply by CODe5753** [reply]
@Denia-park
감사합니다 😊

**Comment by rlaope** [comment]
리스펙합니다.

**Reply by CODe5753** [reply]
@esperar
영광입니다. 감사합니다 🙌

**Comment by sodaMelon** [comment]
안녕하세요~ 글 잘봤습니다!! 저도 애매한 경력(약..2년)으로 이직을 준비하고 있는데요. 혹시 포트폴리오는 작성을 어떤식으로 하셨는지 조언 받을 수 있을까요? 신입때랑 다른게 그 때는 토이프로젝트(본인에게 권한있음)위주로 구성을하였는데, 경력이 쌓인후에는 회사 작업물이다보니 어떤식으로 포폴을 만들어둬야할지 궁금합니다.
+) 이력서 업그레이드 하신것도 잘봤어요!!! ㅋㅋㅋ포켓몬이 3단진화하듯 깔끔하고 세련되게 작성하셨던데 박수를 치고갑니다. 블로그에서 정말 많이배워가요!

**Reply by CODe5753** [reply]
> 경력이 쌓인후에는 회사 작업물이다보니 어떤식으로 포폴을 만들어둬야할지 궁금합니다.

@sodaMelon 님 안녕하세요!
아시다시피, 이력서와 마찬가지로 포트폴리오에도 정답은 없습니다.
아마 작성하셨던 토이프로젝트 포트폴리오와 크게 다르지 않을거라는 생각이 들어요. 
다만, **작업물을 회사에서 관리하다보니 소스코드 첨부나 이미지 첨부 등이 어려울 수 있는데요**. (어쩔 수 없지요 😭)
저의 경우 이력서에 한 줄 정도 간단하게 작성 하셨다면 포트폴리오에서는 그 한 줄에 대해 조금 더 구체적으로 작성하는 편을 선호합니다.
특히 아래 항목을 중요시 여기는데요.

- 트러블 슈팅 과정(배경, 초동 조치, 원인 분석, 영구 조치, 회고)을 자세하게 써두는 것도 좋아요
- 중요기능의 경우 `왜 이렇게 구현했는지`에 대해 작성하는데 비즈니스적인 관점보다는 코드 관점을 선호해요(도메인이 다르면 비즈니스 설명하는데 굉장히 많은 노력을 기울여야 하거든요)
- 이 프로젝트를 진행하면서 배운 점과 그 배운 점으로 다른 곳에서 어떻게 기여(또는 변화, 적용)했는지

포트폴리오도 얼마나 거창하게 했는가 보다는 **이 지원자는 평소 어떤 사고를 갖고 개발에 임하는 지**, **의사 결정 과정에서 중요시 하는 것은 무엇인지**를 보기 때문에 좀 더 구체화 된 이력서라고 생각하면 편할 것 같습니다

도움이 되셨을 지 모르겠네요!

이력서 포스팅에 대한 좋은 피드백도 너무나 감사드립니다 힘이 나는군요!! 🙏🏻
궁금한 점이 있으시다면 언제든 편히 글 남겨주시고, 앞으로도 도움이 될 만한 글을 쓰도록 하겠습니다!
감사해요 ☺️

**Comment by minsu7893** [comment]
안녕하세요!! 

카카오뱅크 준비하면서 많은 도움받았습니다!

저는 계정계개발자포지션 준비중이며 다음주 최종면접을 앞두고있습니다!!

혹시 경영진면접때 인성관련된 어떤 질문을 받았는지 여쭤봐도 될까요~? 

꼭 사내에서 뵀으면 하네요 ㅎㅎ 감사합니다!!

**Reply by CODe5753** [reply]
@minsu7893 안녕하세요 제가 확인이 너무 늦었군요 ㅠㅠ 
이미 면접을 보셨겠지만 사실 인터넷에서도 흔히 찾아볼 수 있는 그런 질문들과 개인에게 맞춰진 질문들이 들어왔답니다 ㅎㅎ 
면접 결과는 어떠셨나요~?
좋은 결과가 있으면 좋을텐데 말이죠!

**Comment by hj970207** [comment]
안녕하세요!!
혹시 경영진 합격 문자에 있는 투두 리스트는 어떤게 있나요??
경영진 합격 발표 전에 신용조회 동의 문자를 받으시고 제출하신걸까요??
마지막으로 입사일은 최종합격 후 몇일 이런식으로 조율하셨을까요??

**Reply by CODe5753** [reply]
안녕하세요~! 시간이 많이 지난 후에 답변을 드리는거지만 비슷하게 **궁금하신 분들이 계실까봐 답글 남겨봅니다.**  
투두 리스트엔 서류 제출이 다였던 것으로 기억합니다.   
미리 준비할 것이 없고 연락 받고 나서 차근차근 준비해도 무방했어요~  
신용조회 동의 문자 받고 제출했던 것 같아요!  
입사일은 제가 원하는 날짜를 말씀드리면 회사에서 조율을 도와주십니다!

**Comment by parkchangeon** [comment]
안녕하세요. 좋은 글 잘 봤습니다!
궁금한게 있어요.

처우 협의에 2주 걸렸다고 적어두셨는데 관련 서류 보내고 응답 오는데 1주일정도 걸렸을 것 같고
나머지 1주는 담당자와 핑퐁을 하셨나요?
처우 협의 2주동안 어떤 내용이 오고 갔는지, 어떤 일이 있었는지 궁금합니다!

**Comment by hrkimm1** [comment]
[제로베이스] 안녕하세요:) 카카오뱅크에서의 합격 여정 함께 나눠주실 수 있을까요?

안녕하세요, 승현님 :)
저는 제로베이스에서 커리어 교육 관련 기획을 담당하고 있는 김혜리입니다.

저희 과정에서는 취업 준비생분들이 실제 합격에 한 걸음 더 다가갈 수 있도록 매월 최신 합격자분들의 자료를 수집·가공하여 학습 자료로 제공하고 있습니다.

이에 이번에 합격하신 합격 서류 (이력서, 자소서, 포트폴리오) 공유가 가능하실지 정중히 여쭙습니다. 공유해주시는 합격 경험은 익명화되어 취준생분들에게 양질의 정보로 제공될 예정인데요.

공유해주신 자료는 개인정보를 철저히 익명화하여 제공되며, 그동안 삼성·네이버·카카오·신한은행 등 주요 기업들의 합격 사례가 안전하게 공유된 바 있습니다. 지금까지 단 한 건의 이슈도 발생하지 않았음을 말씀드립니다.

공유해주신 자료에 대한 리워드로 최소 10만원부터 최대 30만원까지 지급이 가능하며, 자세한 내용은 메일 주소 전달주시면 안내드리겠습니다. 긍정적인 검토 부탁드립니다 🙏🏼

감사합니다.
